spec/examples/teacher_training_api/single_provider_response.json

```diff
@@ -17,7 +17,9 @@
       "street_address_2": "1st floor, 86 Long Rd",
       "train_with_disability": "We are committed to supporting trainees with disabilities and other needs, so please let us and the accrediting provider know if you would like us to make any adjustments to support you.",
       "train_with_us": "We offer both primary and secondary training in a wide range of subjects and work with a collection of schools in the local area. We offer diverse teaching placements, ensuring trainees have experience in different types of school environments.",
-      "website": "http://www.teamworkstsa.org/home-page-s-c-i-t-t-teacher-training/"
+      "website": "http://www.teamworkstsa.org/home-page-s-c-i-t-t-teacher-training/",
+      "latitude": "51.498160",
+      "longitude": "-0.129900"
```

Comment:
This JSON was copy-pasted from the API docs https://api.publish-teacher-training-courses.service.gov.uk/api-reference.html#recruitment_cycles-year-providers-provider_code-get-responses-examples — if these fields aren't there they won't be in the response from the API

Reply:
I think this is the resource we need: https://api.publish-teacher-training-courses.service.gov.uk/api-reference.html#recruitment_cycles-year-providers-provider_code-courses-course_code-locations-get-responses

Reply:
I've got a pr open here https://github.com/DFE-Digital/teacher-training-api/pull/1671 which adds the lat/long and updates the documentation. This is for the providers lat/long, not the sites. Although, by the look of it, i also need to add lat/long for sites.